website/src/pages/en/resources/claud-mcp.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Claude MCP
+---
+
+This guide walks you through configuring Claude Desktop to use The Graph ecosystem's Model Context Protocol (MCP) resources: Token API and Subgraph. These integrations allow you to interact with blockchain data through natural language conversations with Claude.
```

Comment:
I wonder if "Model Context Protocol (MCP)" should link somewhere that explains the concept? Maybe https://modelcontextprotocol.io/?